<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackSysSettingsCutsForm</class>
 <widget class="QWidget" name="TrackSysSettingsCutsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_cuts_momentum">
     <property name="title">
      <string>Momentum cuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_cut_minpt">
            <property name="text">
             <string>Min</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_minpt_gev">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> GeV</string>
            </property>
            <property name="maximum">
             <double>999.990000000000009</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_cut_maxpt">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_maxpt_gev">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> GeV</string>
            </property>
            <property name="maximum">
             <double>999.990000000000009</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_momtype">
          <item>
           <property name="text">
            <string>Pt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>P</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VP1EtaPhiCutWidget" name="etaPhiCutWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reconstructed tracks</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_lala">
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_cut_nprecisionhits_muon">
          <property name="text">
           <string>Require Precision Muon hits:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_cut_nhits_sct">
          <property name="toolTip">
           <string>Hide tracks without this many SCT hits. Only applies to tracks with ID information.</string>
          </property>
          <property name="text">
           <string>Require SCT hits:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_cut_nhits_trt">
          <property name="toolTip">
           <string>Hide tracks without this many TRT hits. Only applies to tracks with ID information.</string>
          </property>
          <property name="text">
           <string>Require TRT hits:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_requireDetectorElement">
          <property name="text">
           <string>Require Detector Element</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_cut_nhits_muon"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_cut_nhits_sct">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_cut_nhits_muon">
          <property name="text">
           <string>Require Muon hits:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_cut_nhits_pixel">
          <property name="toolTip">
           <string>Hide tracks without this many pixel hits. Only applies to tracks with ID information.</string>
          </property>
          <property name="text">
           <string>Require Pixel hits:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBox_cut_nprecisionhits_muon"/>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_cut_nhits_pixel"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_cut_nhits_trt"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_detectorElementId">
          <property name="toolTip">
           <string>Only show tracks which contain this detector element. So will not select tracks with no parameter on this detector element.</string>
          </property>
          <property name="placeholderText">
           <string>Identifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ID requirements only apply to tracks with ID hits (i.e. not MS standalone tracks)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_cuts_truth_tracks">
     <property name="title">
      <string>Truth tracks</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_cut_truthtracks_creationvertexinIR">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Secondaries are particles based solely on simhits, or particles with a creation vertex outside the interaction region (|r|&amp;lt;2.8cm, |z|&amp;lt;50cm).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Require creation vertex at IR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_cut_truthtracks_excludeneutrals">
          <property name="text">
           <string>Exclude neutrals</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_cut_truthtracks_excludebarcode0">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Some simhits will be due to secondary particles without an entry in an HepMC record. Such simhits will be marked as coming from a particle with barcode=0.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Exclude simhits with barcode 0</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_vertexAssociated">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>The vertex system must be enabled for this to be usable.</string>
     </property>
     <property name="text">
      <string>Only show Vertex associated tracks</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1EtaPhiCutWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1EtaPhiCutWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
